subsys/net/lib/aws_fota/include/aws_fota_json.h

```diff
@@ -27,10 +27,34 @@ extern "C" {
  */
 #define STATUS_MAX_LEN (12)
 
-int aws_fota_parse_notify_next_document(char *job_document,
-		u32_t payload_len, char *job_id_buf, char *hostname_buf,
-		char *file_path_buf);
+/**
+ * @brief Parse a given AWS IoT Job Execution JSON object.
+ *
+ * @param[in] job_document JSON formated string of an AWS Job execution
```

Comment:
```suggestion
 * @param[in] job_document JSON formated string of an AWS JobExecution data type
```

Should we add a link here?
https://docs.aws.amazon.com/iot/latest/developerguide/jobs-api.html#jobs-mqtt-job-execution-data

Reply:
This is not a job execution but a Describe job execution Response payload so it's not the job execution data type per say.

Reply:
added link